operator/src/main/resources/scripts/stopServer.sh

```diff
@@ -60,23 +64,23 @@ export SHUTDOWN_TIMEOUT_ARG=${SHUTDOWN_TIMEOUT:-30}
 export SHUTDOWN_IGNORE_SESSIONS_ARG=${SHUTDOWN_IGNORE_SESSIONS:-false}
 export SHUTDOWN_TYPE_ARG=${SHUTDOWN_TYPE:-Graceful}
 
-trace "Before stop-server.py [${SERVER_NAME}] ${SCRIPTDIR}" &>> /weblogic-operator/stopserver.out
-${SCRIPTPATH}/wlst.sh /weblogic-operator/scripts/stop-server.py &>> /weblogic-operator/stopserver.out
-trace "After stop-server.py" &>> /weblogic-operator/stopserver.out
+trace "Before stop-server.py [${SERVER_NAME}] ${SCRIPTDIR}" &>> ${STOP_OUT_FILE}
+${SCRIPTPATH}/wlst.sh /weblogic-operator/scripts/stop-server.py &>> ${STOP_OUT_FILE}
+trace "After stop-server.py" &>> ${STOP_OUT_FILE}
 
 # at this point node manager should have terminated the server
 # but let's try looking for the server process and
 # kill the server if the process still exists,
 # just in case we failed to stop it via wlst
 pid=$(jps -v | grep " -Dweblogic.Name=${SERVER_NAME} " | awk '{print $1}')
 if [ ! -z $pid ]; then
-  echo "Killing the server process $pid" &>> /weblogic-operator/stopserver.out
+  echo "Killing the server process $pid" &>> ${STOP_OUT_FILE}
   kill -15 $pid
 fi
 
-touch /weblogic-operator/doShutdown
+touch /tmp/doShutdown
```

Comment:
If there was trouble writing /weblogic-operator/doShutdown (presumably because this directory is formed as part of a read only config-map), why isn't there some issue with /weblogic-operator/pid that's referenced in the next line?